bycycle/plts/features.py

```diff
@@ -0,0 +1,129 @@
+"""Plot cycle features."""
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from neurodsp.plts.utils import savefig
+
+###################################################################################################
+###################################################################################################
+
+@savefig
+def plot_feature_hist(feature, burst_param, bins='auto', ax=None, **kwargs):
+    """Plot a histogram for a cycle feature.
+
+    Parameters
+    ----------
+    feature : pandas.DataFrame or 1d array
+        Dataframe output from :func:`~.compute_features` or a 1d array.
+    burst_param : str
+        Column name of the parameter of interest in ``df``.
+    bins : int or string, optional, default: 'auto'
+        The number of bins or binning strategy string,
+        as specified in matplotlib.pyplot.hist.
+    ax : matplotlib.Axes, optional, default: None
+        Figure axes upon which to plot.
+    **kwargs
+        Keyword arguments to pass into `matplotlib.pyplot.hist`.
+
+    Notes
+    -----
+    Default keyword arguments include:
+
+    - ``xlabel``: str, default: ``burst_param``
+    - ``figsize``: tuple of (float, float), default: (10, 10)
+    - ``color``: str, default: 'k'
+    - ``xlim``: tuple of (float, float), default: None
+    - ``fontsize``: float, default: 15
+    - ``alpha``: float, deafult: .5
+
+    """
+
+    # Limit dataframe to bursts
+    if isinstance(feature, pd.core.frame.DataFrame):
+        feature = feature[feature['is_burst']][burst_param]
+
+    # Optional keyword args
+    figsize = kwargs.pop('figsize', (5, 5))
+    color = kwargs.pop('color', 'k')
+    xlabel = kwargs.pop('xlabel', burst_param)
+    xlim = kwargs.pop('xlim', None)
+    fontsize = kwargs.pop('fontsize', 15)
+    alpha = kwargs.pop('alpha', .5)
+
+    if ax is None:
+        fig, ax = plt.subplots(figsize=figsize)
+
+    # Axis style
+    ax.set_xlabel(xlabel, size=fontsize)
+    ax.set_ylabel('# cycles', size=fontsize)
+
+    if xlim:
+        ax.set_xlim(xlim)
+
+    ax.hist(feature, bins=bins, color=color, alpha=alpha, **kwargs)
+
+
+@savefig
+def plot_feature_catplot(df, burst_param, group_by=None, ax=None, **kwargs):
+    """ Plot a catplot for a cycle feature.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame.
+        Dataframe output from :func:`~.compute_features`.
+    burst_param : str
```

Comment:
the DF params aren't necessarily burst params, right? Should this just be called `param`, or maybe `label` or `param_label`?